src/cow-react/modules/swap/services/ethFlow/steps/calculateUniqueOrderId.ts

```diff
@@ -11,9 +13,25 @@ export interface UniqueOrderIdResult {
   orderParams: PostOrderParams // most cases, will be the same as the ones in the parameter, but it might be modified to make the order unique
 }
 
+function incrementFee(params: PostOrderParams): PostOrderParams {
+  const nativeCurrency = params.feeAmount?.currency
+
+  if (!nativeCurrency) {
+    throw new Error('Missing currency for Eth Flow Fee') // Not a realistic case, just to make TS happy
+  }
+
+  const oneWei = CurrencyAmount.fromRawAmount(nativeCurrency, 1)
+  return {
+    ...params,
+    feeAmount: params.feeAmount?.add(oneWei), // Increment fee by one wei
+    sellAmountBeforeFee: params.sellAmountBeforeFee?.add(oneWei),
```

Comment:
Shouldn't you remove instead of adding to the sell amount here?

This causes the user to sell more than what initially planned in the UI, no?

Reply:
`sellAmountBeforeFee` is not a thing when posting orders. (only for quoteing).

Reply:
Also, conceptually, `sellAmountBeforeFee` should stay the same (that is the total `sellAmount + feeAmount` when bumping fees.

Reply:
Yep! thanks for pointing it out!